### PR TITLE
Shametable filters and layout

### PR DESCRIPTION
**Pull request by mafiuss**
Fixed the #8 and #27 issues

### CODE REVIEW

File: wizecatFrontend/opinionated/src/wizecat-timer/wizecat-timer.coffee [hunk 1]
```diff
@@ -18,11 +18,15 @@ class wizecatTimer extends Polymer.Element
   ready: ->
     super.ready()
     try
-      @socket = window.io('http://localhost:3000')
+      @socket = window.io('http://wize.cat/')
```
Comment (mariogillazaro) [inline wizecatFrontend/opinionated/src/wizecat-timer/wizecat-timer.coffee]: Why is this necessary?

File: wizecatFrontend/opinionated/src/wizecat-timer/wizecat-timer.coffee [hunk 2]
```diff
@@ -18,11 +18,15 @@ class wizecatTimer extends Polymer.Element
...
     catch error
       console.log 'error connecting to the socket'
 
   _nameChanged: (newName, oldName) ->
-    @socket.emit('catping', {name: newName})
+    return
+    # @socket.emit('catping', {name: newName})
```
Comment (mariogillazaro) [inline wizecatFrontend/opinionated/src/wizecat-timer/wizecat-timer.coffee]: We should avoid hosting commented code

File: wizecatFrontend/opinionated/src/wizecat-timer/wizecat-timer.coffee (hunk 1 above)
Comment (sabs231) [reply wizecatFrontend/opinionated/src/wizecat-timer/wizecat-timer.coffee]: That's the line which connects the client with the server on socket.io

File: wizecatFrontend/opinionated/src/wizecat-timer/wizecat-timer.coffee (hunk 2 above)
Comment (mafiuss) [reply wizecatFrontend/opinionated/src/wizecat-timer/wizecat-timer.coffee]: Agreed will remove

File: wizecatFrontend/opinionated/src/wizecat-timer/wizecat-timer.coffee (hunk 1 above)
Comment (mafiuss) [reply wizecatFrontend/opinionated/src/wizecat-timer/wizecat-timer.coffee]: It's necessary to use the domain name because when the browser opens this socket localhost for this browser is not wize.cat socket is the localhost.
